cli/ts/commands/deployPoll.ts

```diff
@@ -43,38 +51,56 @@ export const deployPoll = async (
     logError("Please provide a VkRegistry contract address");
   }
 
-  const vkRegistry = vkRegistryAddress ? vkRegistryAddress : vkRegistryContractAddress;
+  const vkRegistry = vkRegistryAddress || vkRegistryContractAddress;
 
-  const _maciAddress = readContractAddress("MACI");
-  if (!_maciAddress && !maciAddress) {
+  const maciContractAddress = readContractAddress("MACI");
+  if (!maciContractAddress && !maciAddress) {
     logError("Please provide a MACI contract address");
   }
 
-  const _maci = maciAddress ? maciAddress : _maciAddress;
+  const maci = maciAddress || maciContractAddress;
 
   // required arg -> poll duration
-  if (pollDuration <= 0) logError("Duration cannot be <= 0");
+  if (pollDuration <= 0) {
+    logError("Duration cannot be <= 0");
+  }
   // require arg -> max messages
```

Comment:
typo: required
btw, do we need these comments?

Reply:
This is not related to this PR.